src/main/java/br/com/sbs/cubatech/validation/Validator.java

```diff
@@ -8,9 +8,12 @@
     public static final String MESSAGE_INTERVAL_VALIDATION = "Time to finish, needs to be between 1 or 20 hours.";
 
     public static void notEmptyOrNull(String text, String error){
-        if(text == null || text.isBlank()) {
+        if(text.isBlank()) {
             throw new IllegalArgumentException(error + MESSAGE_NOT_EMPTY_OR_NULL);
         }
+        if (text.equals(null)) {
```

Comment:
Assim evitar o npe porque o `text` pode ser nulo
```suggestion
        if (text == null) {
```